core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java

```diff
@@ -57,6 +69,9 @@ public synchronized String getPropertiesPath() {
           + " set in Zookeeper). Expected format: -o <key>=<value>")
   private List<String> overrides = new ArrayList<>();
 
+  // holds property overrides that may be generated from deprecation:
+  private static List<String> deprecated_overrides = new ArrayList<>();
```

Comment:
Those legacy options are more for client-specific config. But, throwing the exception might be a good model to follow instead of trying to translate.

Reply:
So...I believe we are saying that the best path forward at this time is to do something similar to how ClientOpts is handling legacy (or dropped) options.  No deprecation and auto setting properties.  This definitely simplifies things and keeps the code tidier.   Since these are command line options, dropping them instead of "deprecating" them is not so bad since they are easily updated by the customer vs. "deprecated" java code.  The worst case being that customer generated scripts may need to be updated.

Reply:
> This definitely simplifies things and keeps the code tidier. Since these are command line options, dropping them instead of "deprecating" them is not so bad since they are easily updated by the customer vs. "deprecated" java code.

I think you just convinced me that this is the right course of action.

Reply:
farout! ;)